.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Fixes #issue(s).
```

Comment:
Maybe more description here, to make sure that it's understood as a placeholder and this would be deleted on a feature request.